docs/topics/Data-Verification.md

```diff
@@ -0,0 +1,62 @@
+<note>
+We do not ask for beatmap IDs to be separated by round during submission, since this would require much more manual work and round names are often not standardized between tournaments. This means that a warmup game may very rarely be counted as a verified game if it was played on a map from a different round in the same tournament; if you notice this, please submit a ticket to the o!TR team. 
+</note>
+</step>
+<step>
+Finally, any remaining <code>PreVerified</code> data is marked as <code>Verified</code>, and any remaining <code>PreRejected</code> data is marked as <code>Rejected</code>.
+</step>
+</procedure>
+
+<tip>
+Match data involving players who were restricted, including those who were tournament banned for foul play, is handled by the system just like any other match data. However, restricted players will not appear on any rating leaderboards. Depending on the influence of such players on the match, the o!TR team may include or exclude such games or matches from verification on a case-by-case basis.
+</tip>
+
+## Modifying submitted data
+
+The o!TR admins have the ability to manually modify match data in the database, including game scores, tournament names, and match win records. (Note that this is different from simply changing the verification status of a game score, game, match, or tournament.) Any such events will always be clearly documented in admin notes and made publicly viewable. Below is a non-exhaustive list of cases where this may be done, pending feature development:
+
+* Renaming a tournament for consistency with past or future iterations,
+* Correcting the EZ (or other mod) multipliers specified in a particular tournament's ruleset by manually changing the appropriate game scores,
+* Adding a new game score to correct a lobby size mismatch if (1) the player disconnected mid-game but provided a local replay or records of an on-stream score and (2) that data was actually used as the source of truth for tournament proceedings.
+* Assigning team rosters to a 1v1 match played in the `HeadToHead` team mode so that the match may be converted to `TeamVS`,
+* Removing "phantom mods" that mistakenly appear in game scores (for example, players may mistakenly appear to play a map with the DT mod if the previous map was played with DT),
+* Merging two matches into one if a match was split across two MP links due to technical difficulties.
+
+Any modifications of this kind, as well as any changes to verification status detailed above, will be recorded in audit logs. These logs will include the time the change was made, the user who performed it, and the state of the object before and after the change; thus, any manual edits will be trackable and reversible if needed. While these audit logs will not be available in the [public replicas](Data.md#public-replicas) (since they are attached to specific users), more transparent record-keeping is in development.
```

Comment:
```suggestion
Any modifications of this kind, as well as any changes to verification status detailed above, will be recorded in audit logs. These logs include the time the change was made, the user who performed it, and the state of properties before and after the change; thus, any manual edits will be trackable and reversible if needed. While these audit logs will not be available in the [public replicas](Data.md#public-replicas) (since they are attached to specific users), more transparent record-keeping is in development.
```